x-pack/plugins/observability/public/update_global_navigation.tsx

```diff
@@ -37,15 +42,18 @@ export function updateGlobalNavigation({
           ...link,
           navLinkStatus: someVisible ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden,
         };
-      case 'slos':
+      case 'rules':
         return {
           ...link,
           navLinkStatus: someVisible ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden,
         };
-      case 'rules':
+      case 'slos':
         return {
           ...link,
-          navLinkStatus: someVisible ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden,
+          navLinkStatus:
+            someVisible && capabilities[sloFeatureId].read
+              ? AppNavLinkStatus.visible
+              : AppNavLinkStatus.hidden,
```

Comment:
💬 Keeping the same logic as Cases: We want apm or logs or metrics or uptime to be enabled, and the slo feature read capabilities to show the slo menu entry